test/scrabble_test.rb

```diff
@@ -5,8 +5,17 @@
 
 class ScrabbleTest < Minitest::Test
```

Comment:
I would also like to see a test for an empty string and nil being passed in as arguments.